proto/vtctlservice.proto

```diff
@@ -191,4 +191,11 @@ service Vtctld {
   // parameters. Empty values are ignored. If the alias does not exist, the
   // CellsAlias will be created.
   rpc UpdateCellsAlias(vtctldata.UpdateCellsAliasRequest) returns (vtctldata.UpdateCellsAliasResponse) {};
+  // ValidateSchemaKeyspace will diff the schema from all the tablets in
+  // the keyspace.
+  rpc ValidateSchemaKeyspace(vtctldata.ValidateSchemaKeyspaceRequest) returns (vtctldata.ValidateSchemaKeyspaceResponse) {};
+  // ValidateSchemaShard will diff the schema from all the tablets in the shard.
+  rpc ValidateSchemaShard(vtctldata.ValidateSchemaShardRequest) returns (vtctldata.ValidateSchemaShardResponse) {};
+  // ValidateVSchema compares the schema of each primary tablet in "keyspace/shards..." to the vschema and errs if there are differences
```

Comment:
This is too close to the details of the old cli, and will be more confusing to grpc clients that don't come from that specific use case. We probably want something more generic (like the other two you added), something like:

> ValidateVSchema compares the schema of the primary in each specified shard to the vschema. The RPC fails if there are differences.